MAS/MAS.py

```diff
@@ -873,6 +885,7 @@ def Analyse_RowAs_Kmeans(self):
                 
             print("Rows to_delete", to_delete)
             nb_to_delete = len(to_delete)
+            to_delete = [item for sublist in to_delete for item in sublist]
```

Comment:
With my suggested changes, this line becomes unecessary and can be deleted. Rather, it introduces a bug for no reason😉